pandas/tests/io/json/test_compression.py

```diff
@@ -41,6 +41,7 @@ def test_read_zipped_json(datapath):
 
 @td.skip_if_not_us_locale
 @pytest.mark.single_cpu
+@pytest.mark.network
```

Comment:
These test markers are extra, but I noticed locally that these would hang (they all make s3 calls)